src/services/identity/local.ts

```diff
@@ -30,20 +29,18 @@ const {
   ISSUER_DID
 } = process.env
 
-export class LocalIdentity implements IIdentity {
-  agent?: VeramoAgent
-  privateStore?: AbstractPrivateKeyStore
-
+export class LocalIdentity extends DefaultIdentity {
   constructor() {
```

Comment:
Am I right that such constructor is the same as for all child classes also? If yes - let's move it to the default class

Reply:
We cannot move it to default constructor of `DefaultIdentity` class because when `DefaultIdentity` calls own constructor then Virtual Table is not fully complete, and it means: if I put constructor implementation to `DefaultIdentity` class default constructor, then inside `DefaultIdentity` constructor will call unimplemented `DefaultAgent::initAgent()` method.

Reply:
Done ✅